README.md

```diff
@@ -2,7 +2,7 @@
 
 [![CI status](https://github.com/babylonhealth/orbit-mvi/workflows/Android%20CI/badge.svg)](https://github.com/babylonhealth/orbit-mvi/actions)
 [![codecov](https://codecov.io/gh/babylonhealth/orbit-mvi/branch/master/graph/badge.svg)](https://codecov.io/gh/babylonhealth/orbit-mvi)
-[![Download](https://api.bintray.com/packages/babylonpartners/maven/orbit/images/download.svg?version=3.4.0) ](https://bintray.com/babylonpartners/maven/orbit/3.4.0/link)
+[![Download](https://api.bintray.com/packages/babylonpartners/maven/orbit/images/download.svg?version=3.4.0)](https://bintray.com/babylonpartners/maven/orbit/3.4.0/link)
```

Comment:
This version is no longer the latest is it?

Reply:
There's `3.5.0` but it's a pre-release that we haven't tested yet.